### PR TITLE
Filters in Map and Fake Data for Descriptors

### PR DESCRIPTION
**Pull request by nnarang7**
- Added filters and advanced filters to map interface
- Added fake data for descriptors tables

### CODE REVIEW

**Comment by weronica** [comment]
Can you add screenshots of the UI changes that you made @nnarang7?

**Comment by armant** [comment]
@dzhang55

File: app/main/views.py [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+    if len(all_filters) > 3:
+    	top_filters.append(all_filters[0].name)
+    	top_filters.append(all_filters[1].name)
+    	top_filters.append(all_filters[2].name)
+    	for index in range(3, len(all_filters)):
+    		advanced_filters.append(all_filters[index].name)
+    else:
+    	for filt in all_filters:
+        	top_filters.append(filt.name)
+    return render_template('main/index.html', top = top_filters, advanced = advanced_filters)
```
Comment (dzhang55) [inline app/main/views.py]: This is super minor, but I think top and advanced should still be named top_filters and advanced_filters just so it's a bit more clear in the index.html file.

Also there's no newline at the end of the file.

File: app/assets/scripts/vendor/map.js [hunk 2]
```diff
@@ -101,6 +101,14 @@
             $("#list").show();
         });
 
+      var advanced_options_button = $("#advanced-options-button");
+      var advanced_options_checkboxes = $("#advanced-options-checkboxes");
+      advanced_options_button.click(function() {
+        console.log("did i do something");
```
Comment (dzhang55) [inline app/assets/scripts/vendor/map.js]: nit: print statements

File: app/main/views.py [hunk 3]
```diff
@@ -1,7 +1,21 @@
 from flask import render_template
 from . import main
+from ..models import Descriptor
 
 
-@main.route('/')
+@main.route('/', methods=['GET'])
 def index():
-    return render_template('main/index.html')
+    print "getting filters"
```
Comment (dzhang55) [inline app/main/views.py]: nit: print statements

**Pull request closed without merging**